packages/dds/tree/src/test/shared-tree/fuzzEditGenerator.ts

```diff
@@ -65,6 +66,11 @@ export interface TreeContext {
 	treeIndex: number;
 }
 
+export interface PathWithNodeRange {
+	path: UpPath;
```

Comment:
Nit: it could be nice to clarify that this is the path to the first node. You could either rename the field or put a doc comment on it. Not a big deal either way.

Reply:
updated path name